LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2020, Brookhaven Science Associates, Brookhaven National Laboratory
```

Comment:
Last I saw it should only be "Brookhaven National Laboratory" and we have been using that in repositories.

Reply:
I copied this from the SRX one that is linked at issue.https://github.com/NSLS-II-SRX/profile_collection/pull/16

Reply:
Then that is wrong too!